src/components/Icons/ChevronDown.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const ChevronDownIcon = ({ size, color, ...props }) => (
```

Comment:
Think about introducing this as a single `<ChevronIcon />` component that takes a `direction` prop.